### PR TITLE
after two quiet years it is time to add just one option to boring: su…

### PR DESCRIPTION
**Pull request by boutell**
…pport for an end of options marker. This is a common convention in shell commands, for more information see: https://unix.stackexchange.com/questions/11376/what-does-double-dash-mean

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# boring
+# `boring`
```
Comment (abea) [inline README.md]: We don't normally format titles in code, as in sanitize-html. It reveals that the thing  might not have been given a human n the title of the thing. 

[Chalk](https://www.npmjs.com/package/chalk) is one example. It's been given the readable name of "Chalk," which looks like a normal English proper noun. webpack, is officially lowercase (as boring was), and is also the command, but isn't formatted as code.

TLDR; we should leave those as it was or capitalize them outside of actual code examples.

File: boring.js [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
   const args = process.argv.slice(2);
   const result = {
     _: []
   };
+  let optionsEnded = false;
   for (let i = 0; (i < args.length); i++) {
-    let matches = args[i].match(/^--([^\s\=]+)=(.*)$/);
-    if (matches) {
-      result[matches[1]] = matches[2];
+    if (options.end && (args[i] === options.end)) {
```
Comment (abea) [inline boring.js]: Is it better for this to be an option option, or to define that double dashes are the ender indicator? If it's common elsewhere it seems a reasonable ender. Or default to that with `true` and allow an alternate if the value is a string.

**Comment by boutell** [comment]
kk, howzabout now?

**Review by abea — APPROVED** [review]
